Slim/Web/Cometd.pm

```diff
@@ -658,7 +660,7 @@ sub sendResponse {
 	else {
 		# For CLI, don't send anything if there are no events
 		if ( scalar @{$out} ) {
-			sendCLIResponse( $httpClient, $httpResponse, $out );
+			sendCLIResponse( $httpClient, $out );
```

Comment:
What about this change? I see that `sendCLIRequest()` would only accept two parameters. So this change makes sense - except that you're now passing different data to it, not less. How would that now work if it wasn't broken before?

Reply:
Ok, I believe you gave the answer already: that code was never reached. So essentially this should have been supported from the early days. But nobody used it or understood it, therefore nobody complained about it being broken?...

Could you please add a few lines to the CLI documentation, too? Explaining how to use the CLI in JSON mode:

https://github.com/LMS-Community/lms-community.github.io/blob/main/docs/reference/cli/introduction.md
https://github.com/LMS-Community/lms-community.github.io/blob/main/docs/reference/cli/using-the-cli.md

Reply:
Thanks for the review and patience on this Michael!

I meant to note that the `$httpClient` variable is unfortunately named here for CLI requests but I left it as-is to minimize the changes.

I understand your concern on breaking existing HTTP functionality. Looking at my sodface.com web server logs, I have  6 or 7 people using my 8.5.3 LMS package on Alpine. I could push this change to my package first and post in the Alpine forum thread about it, that way there's at least a little more testing done? Or if you consider 9.x testing already then maybe that's not necessary?

Reply:
Yes, after better understanding the change I'd be willing to merge to LMS9. As it's in such a central piece of LMS, any issue should show up quickly, I think.

Reply:
Wanted to clarify this PR only addresses:  
`/meta/handshake` to get a clientID assigned, and  
`/slim/request` to do one off commands/queries with json responses

Some of the other Cometd functions are probably still broken for CLI clients, like subscribe and unsubscribe. I was trying to keep this simple and restore basic /slim/request function. Is that ok for now?

Reply:
IMHO this PR is about getting JSON structured data over CLI, not about implementing all cometd features in CLI. cometd is a transfer protocol, like the telnet style CLI's connection is. So we're good. You might be the only one to use the CLI this way anyway. Because: why use it when you can use JSON/RPC? 😝